<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>271</width>
     <height>471</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pushButton_setlinenum">
        <property name="text">
         <string>设置行数</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox_linenum"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_inittabledata">
      <property name="text">
       <string>初始化表格数据</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="pushButton_insertline">
        <property name="text">
         <string>插入行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_addline">
        <property name="text">
         <string>添加行</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_delectline">
      <property name="text">
       <string>删除当前行</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="pushButton_autolineH">
        <property name="text">
         <string>自动调节行高</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_autorowW">
        <property name="text">
         <string>自动调节列宽</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QPushButton" name="pushButton_readtoText">
      <property name="text">
       <string>读取表格内容到文本</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QCheckBox" name="checkBox_tableeditable">
      <property name="text">
       <string>表格可编辑</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QCheckBox" name="checkBox_linebgcolor">
      <property name="text">
       <string>间隔行底色</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="checkBox_HshowtableHeader">
      <property name="text">
       <string>显示水平表头</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="checkBox_VshowtableHeader">
      <property name="text">
       <string>显示垂直表头</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QRadioButton" name="radioButton_chooseline">
      <property name="text">
       <string>行选择</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QRadioButton" name="radioButton_choosetableitem">
      <property name="text">
       <string>单元选择</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_settableHeader">
      <property name="text">
       <string>设置水平表头</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_clearplainedit">
      <property name="text">
       <string>清空文本</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>20</y>
     <width>431</width>
     <height>471</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableWidget" name="tableWidget">
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <row/>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>新建列</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="widget_status" native="true">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>509</y>
     <width>711</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <action name="actionclear_edit">
   <property name="text">
    <string>clear_edit</string>
   </property>
   <property name="menuRole">
    <enum>QAction::NoRole</enum>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
